Sources/containertool/containertool.swift

```diff
@@ -28,112 +28,141 @@ enum AllowHTTP: String, ExpressibleByArgument, CaseIterable { case source, desti
         abstract: "Build and publish a container image"
     )
 
-    @Option(help: "Default registry for references which do not specify a registry")
-    private var defaultRegistry: String?
-
-    @Option(help: "Repository path")
-    private var repository: String?
-
     @Argument(help: "Executable to package")
     private var executable: String
 
-    @Option(help: "Resource bundle directory")
-    private var resources: [String] = []
+    /// Options controlling the locations of the source and destination images
+    struct RepositoryOptions: ParsableArguments {
+        @Option(help: "Default registry for image references which do not specify one")
+        var defaultRegistry: String?
 
-    @Option(
-        help: ArgumentHelp(
-            "[DEPRECATED] Default username, used if there are no matching entries in .netrc. Use --default-username instead.",
-            visibility: .private
-        )
-    )
-    private var username: String?
+        @Option(help: "Destination image reference")
+        var repository: String?
 
-    @Option(help: "Default username, used if there are no matching entries in .netrc")
-    private var defaultUsername: String?
+        @Option(help: "Destination image tag")
```

Comment:
```suggestion
        @Option(help: "The  image tag")
```

(was destination meaningful? is there a difference in how the tool handles local vs. remote for adding tags to the container image?)

Reply:
I think destination is meaningful here because this tag is only applied to the image we are building.   The base image may also have a tag, but that is part of the base image reference.  For instance, in `--from swift:slim`, the repository is `swift` (most images are of the form `username/subrepository` but official images such as `swift` get to be known by one name, like pop stars 🤣) and the tag is `slim`.

Reply:
I've raised #130 to address the relationship between the `--tag` argument and a container image name which already includes a tag.